src/index.ts

```diff
@@ -6,9 +6,3 @@ const ui = SwaggerUI({
   url: 'schemas/fingerprint-server-api.yaml',
   onComplete: () => ui.preauthorizeApiKey('ApiKeyQuery', process.env.PRIVATE_KEY),
 });
-
```

Comment:
Why remove this, did I miss something?

Reply:
Oh I see, we changed the structure and no longer need the separate schema, all good!